tests/nightly/security-internal-apparmor/task.yaml

```diff
@@ -0,0 +1,38 @@
+summary: Check internal vendored apparmor in snapd snap
```

Comment:
Silly question - what is the benefit of this test? Could we test the same in the `snapd-snap` test?

Reply:
This predates the `snapd-snap` test - and infact we already do this and more in the `snapd-snap` test - I will remove it.

Reply:
See dce92ac73e